aws/resource_aws_cloudformation_stack_set.go

```diff
@@ -76,6 +101,15 @@ func resourceAwsCloudFormationStackSet() *schema.Resource {
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"permission_model": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					cloudformation.PermissionModelsServiceManaged,
+					cloudformation.PermissionModelsSelfManaged,
+				}, false),
```

Comment:
Nit: Since this pull request was originally introduced, the AWS SDK Go created functions to retrieve all (modeled) enumeration values. See also: https://github.com/hashicorp/terraform-provider-aws/issues/14601

```suggestion
				ValidateFunc: validation.StringInSlice(cloudformation.PermissionModels_Values(), false),
```